src/components/facet/_facet_group.scss

```
@@ -1,9 +1,25 @@
+$gutterTypes: (
+  gutterExtraSmall: $euiSizeXS,
+  gutterSmall: $euiSizeS,
+  gutterMedium: $euiSize,
+  gutterLarge: $euiSizeL,
+  gutterExtraLarge: $euiSizeXXL,
+);
+
 .euiFacetGroup--horizontal {
   .euiFacetButton {
     height: $euiButtonHeightSmall;
 
-    &:not(:last-of-type) {
-      margin-right: $euiSizeL;
+  }
+}
+@each $gutterName, $gutterSize in $gutterTypes {
```

You'll want to enable a SASS lint plugin for your IDE to catch these spacing issues.
```suggestion
}

@each $gutterName, $gutterSize in $gutterTypes {
```